src/DataStructures/Tensor/Expressions/Evaluate.hpp

```diff
@@ -329,4 +368,58 @@ auto evaluate(const RhsTE& rhs_tensorexpression) {
                                 ~rhs_tensorexpression);
   return lhs_tensor;
 }
+
+/*!
+ * \ingroup TensorExpressionsGroup
+ * \brief Update a LHS tensor to the result of a RHS tensor expression that
```

Comment:
Similar comment, the `update X to Y` phrasing doesn't really make it clear to me when I should use this. In a perfect world people will read the entire documentation, but I don't do that and so can't realistically expect others to either :) The brief needs to highlight _when_ this should be used. E.g. `If the LHS tensor is used in the RHS expression, then use this function to compute the result.` or something like that.

Reply:
Very good point

Reply:
Addressed by new fixup https://github.com/sxs-collaboration/spectre/pull/3842/commits/f4a8355967b0d42817858a11d8927ac71025017c